srcalc/src/test/java/gov/va/med/srcalc/domain/SampleObjects.java

```diff
@@ -170,17 +170,20 @@ public static MultiSelectVariable sampleGenderVariable()
         final List<MultiSelectOption> options = Arrays.asList(
                 new MultiSelectOption("Male"),
                 new MultiSelectOption("Female"));
-        return new MultiSelectVariable(
+        final MultiSelectVariable var = new MultiSelectVariable(
```

Comment:
Let's add the key to all the Variable subclass constructors. Every properly-configured Variable should have a key so we should enforce that through the API.